doc/api/http/request.md

```diff
@@ -181,11 +181,11 @@ http::request({
 
 # BUGS
 
-- It is not possible to search HTTP headers in a case insensitive fashion
+- The `http/ignore_body` setting is not implemented.
 
-- The `Response::response_line` field is always empty
-
-- The `http/ignore_body` setting is not implemented
+- The `Var<Response>` returned by the various callbacks MAY be pointing
+  to `nullptr` and, moreover, there MAY be cases where `Var<Response> response`
+  is pointing to a valid response but `response->request` is `nullptr`.
```

Comment:
Ah, yes, while there, I also updated the documentation removing a couple of sentences that are not accurate anymore and making sure we document the behaviour observed in #1169.